src/pages/billing/Project.js

```diff
@@ -60,7 +60,7 @@ const WorkspaceCardHeaders = Utils.memoWithName('WorkspaceCardHeaders', ({ needs
   ])
 })
 
-const ExpandedInfoRow = Utils.memoWithName('ExpandedInfoRow', ({ title, details, errorMessage }) => {
```

Comment:
Can we confirm that without this it shows up with the right name in the React dev tools?